biscuit-auth/src/token/authorizer.rs

```diff
@@ -143,10 +133,16 @@ impl<'t> Authorizer<'t> {
         let mut blocks = Vec::new();
 
         let authority = token.block(0)?;
-        let block_scope = authority.origins(0, Some(&self.public_key_to_block_id));
 
         let mut authority_origin = Origin::default();
         authority_origin.insert(0);
+
+        let authority_trusted_oringins = TrustedOrigins::from_scopes(
```

Comment:
```suggestion
        let authority_trusted_origins = TrustedOrigins::from_scopes(
```

Reply:
gnnn, i made all the patch telling myself "don't forget to fix the typo when you're done"